src/test/java/com/wikia/webdriver/testcases/discussions/NavigatingTests.java

```diff
@@ -9,12 +9,14 @@
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.discussions.PostsListPage;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.Post;
+import com.wikia.webdriver.elements.mercury.components.discussions.desktop.BackButtons;
```

Comment:
Added spaces before assertions, I believe this comment was made in wrong line? ;)